pyanaconda/localization.py

```diff
@@ -390,6 +390,87 @@ def get_territory_locales(territory):
     return langtable.list_locales(territoryId=territory)
 
 
+def normalize_language(lang):
+    """Normalize the language string to extract the relevant part."""
+    parts = lang.split(".")[0].split("_")
+    return parts[1].lower() if len(parts) > 1 else parts[0]
+
+
+def get_keyboard_layouts_for_language(lang):
```

Comment:
Why not use https://github.com/rhinstaller/anaconda/blob/main/pyanaconda%2Fui%2Fgui%2Fxkl_wrapper.py#L70 ?

I think it provides this functionality already. Doesn't it? 
I would not import it from `XklWrapper` as the class should be removed in the future but this functionality could be moved to the localization module and XklWrapper could import it from here. If it provides what you want.

My point is that the Anaconda GTK UI already lists all the layouts and variants for keyboards so the code has to be already available. I don't think we should be reimplementing it if it is not necessary.